media/css/newsletter/newsletter-developer.scss

```diff
@@ -5,26 +5,40 @@
 @import '../pebbles/includes/lib';
 
 .section-intro {
-    background: url('/media/img/newsletter/developer/heart.svg') top 30px center no-repeat,
-                url('/media/img/newsletter/developer/grid.svg') top left repeat,
-                #0c99d5;
-    @include background-size(200px 199px, 60px 60px);
-    height: 230px;
+    background: url('/media/img/newsletter/developer/hearts.svg') top 30px center no-repeat,
+                url('/media/img/newsletter/developer/bg-image.png') top left no-repeat,
+                #000;
+
+    @include background-size(66px 44px, auto auto);
+    background-position: top 75px center, top center;
+    min-height: 230px;
     width: 100%;
 
+    h1 {
+        color: #fff;
+        font-weight: 100;
+        margin-top: 110px;
+        text-align: center;
+    }
+
     @media #{$mq-phone-wide} {
-        @include background-size(260px 258px, 80px 80px);
+        @include background-size(96px 64px, auto auto);
+        background-position: top 85px center, top center;
         height: 290px;
+        .page-title{
```

Comment:
Nit: space before brace. Also we usually add a blank line before nested rules. (I really need to update our CSS style guide and get it published somewhere we can actually find it...)